### PR TITLE
[159] Create & Style the customer Signup page using css -Frontend

### PR DESCRIPTION
**Pull request by Lillian-1**
Screenshot 2020-12-20 230246
Screenshot 2020-12-20 230322
Screenshot 2020-12-20 230405
Screenshot 2020-12-20 230432
Screenshot 2020-12-20 230456
Screenshot 2020-12-20 230516

### CODE REVIEW

File: submissions/lillianNwaoha/css/signup.css [hunk 1]
```diff
@@ -0,0 +1,320 @@
...
+  font-family: 'Montserrat', sans-serif;
+  overflow-x: hidden;
+}
+nav {
+  background-color: #0066f5;
+  height: 190px;
+  width: auto;
+}
+.logo img {
+  width: 90px;
```
Comment (ToluAkin) [inline submissions/lillianNwaoha/css/signup.css:14]: Only set either of the height or width not both @Lillian-1

File: submissions/lillianNwaoha/css/signup.css [hunk 2]
```diff
@@ -0,0 +1,320 @@
...
+  margin: 40px 0px 0px 20px;
+  border-radius: 10px;
+}
+fieldset {
+  border: 1px solid black;
+  border-radius: 10px;
+  height: 690px;
+  width: 70%;
+  margin-left: 70px;
+  margin-top: 30px;
```
Comment (ToluAkin) [inline submissions/lillianNwaoha/css/signup.css:67]: This was set twice @Lillian-1

File: submissions/lillianNwaoha/css/signup.css [hunk 3]
```diff
@@ -0,0 +1,320 @@
...
+}
+.footer-link {
+  margin-left: 20px;
+  list-style: none;
+  margin-top: 2px;
+}
+.footer-link a {
+  text-decoration: none;
+  font-size: 12px;
+  font-weight: normal;
```
Comment (ToluAkin) [inline submissions/lillianNwaoha/css/signup.css:120]: For everywhere you've used normal as the value of an attribute, you should remove it. It is normal by default. @Lillian-1

File: submissions/lillianNwaoha/html-forms/signup.html [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+    </div>
+
+    <div class="form">
+       <p>Already got an account? <a href="#signin">Sign in</a></p>
+    </div>
+    <div class="forms">
+       <form action="">
+       <fieldset>
+               <label for="first name">
+                  First Name</label><br> <input type="text"><br>
```
Comment (ToluAkin) [inline submissions/lillianNwaoha/html-forms/signup.html:36]: Remove all `br` elements in this file and style it with css @Lillian-1

**Review by ToluAkin — CHANGES_REQUESTED** [review]

File: submissions/lillianNwaoha/html-forms/signup.html (hunk 4 above)
Comment (Lillian-1) [reply submissions/lillianNwaoha/html-forms/signup.html]: done

**Comment by Lillian-1** [comment]
I’ve made the changes


Sent from Yahoo Mail for iPhone


On Monday, December 21, 2020, 9:54 AM, ToluAkin <notifications@github.com> wrote:




@ToluAkin requested changes on this pull request.

In submissions/lillianNwaoha/css/signup.css:
> @@ -0,0 +1,320 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: 'Montserrat', sans-serif;
+  overflow-x: hidden;
+}
+nav {
+  background-color: #0066f5;
+  height: 190px;
+  width: auto;
+}
+.logo img {
+  width: 90px;

Only set either of the height or width not both @Lillian-1

In submissions/lillianNwaoha/css/signup.css:
> +  padding-right: -10px;
+  padding-left: -30px;
+  width: 70%;
+}
+
+.forms {
+  margin: 40px 0px 0px 20px;
+  border-radius: 10px;
+}
+fieldset {
+  border: 1px solid black;
+  border-radius: 10px;
+  height: 690px;
+  width: 70%;
+  margin-left: 70px;
+  margin-top: 30px;

This was set twice @Lillian-1

In submissions/lillianNwaoha/css/signup.css:
> +  padding-top: 50px;
+  width: auto;
+}
+.foot {
+  display: block;
+  margin: 0px 60px 0px 30px;
+}
+.footer-link {
+  margin-left: 20px;
+  list-style: none;
+  margin-top: 2px;
+}
+.footer-link a {
+  text-decoration: none;
+  font-size: 12px;
+  font-weight: normal;

For everywhere you've used normal as the value of an attribute, you should remove it. It is normal by default. @Lillian-1

In submissions/lillianNwaoha/html-forms/signup.html:
> +        </ul>
+    </nav>
+
+    <div class="top">
+        <h1>Register your Account</h1>
+        <p>Registering an account gives you access to sweet promo and events available to only registered customers.</p>
+    </div>
+
+    <div class="form">
+       <p>Already got an account? <a href="#signin">Sign in</a></p>
+    </div>
+    <div class="forms">
+       <form action="">
+       <fieldset>
+               <label for="first name">
+                  First Name</label><br> <input type="text"><br>

Remove all br elements in this file and style it with css @Lillian-1

File: submissions/lillianNwaoha/css/signup.css (hunk 1 above)
Comment (adaugochi) [reply submissions/lillianNwaoha/css/signup.css:14]: Kindly fix this comment @ToluAkin dropped of your PR

**Review by adaugochi — CHANGES_REQUESTED** [review]

File: submissions/lillianNwaoha/css/signup.css (hunk 1 above)
Comment (Lillian-1) [reply submissions/lillianNwaoha/css/signup.css:14]: I have fixed this code

**Review by adaugochi — APPROVED** [review]

**Pull request merged**